lib/server/routers/index.js

```diff
@@ -72,7 +73,7 @@ const inheritArgs = (parent, child) => {
 const bindRoute = (http, route, parentPath = '') => {
     const localPath = appendPath(parentPath, route.path, route.pathOption);
     if (route.method && route.controller) {
-        if (process.env.NODE_ENV === 'test') {
+        if (isInTestMode() || isInDevMode()) {
```

Comment:
As I said, I disagree with this.
We need to be able to check that the routes are correctly blocked in development, and I even disagree to make them all public in test mode.
Please remove this change, but the refactoring is otherwise a good idea.

Reply:
removed